public/js/index.js

```diff
@@ -9,6 +9,21 @@ const getNextId = function () {
   return intId + 1
 }
 
+const positionWidgets = function () {
+  $('.widget').each((i, element) => {
+    const objWidget = $(element).data('widget')
+    $(element).position({
+      ...objWidget.position,
+      of: '#widgets',
+      collision: 'none none'
+    })
+  })
+}
+
+$( window ).on( "resize", function() {
```

Comment:
Sometimes you add a SPACE after the open paren, other times you don't. What does the linter have to say? Which JavaScript coding style are we using?

Reply:
our js coding style is very different from other ones I have used, mostly following eslint already there. In this case it didnt pick it up at all not sure why. I removed the spaces. I prefer to not arbitrary space, but in general dont focus too much on whitespace in a language that does respect it. We can set up an auto formatter too.